design-editor/src/system/assistant/assistant-code-generator.js

```diff
@@ -130,8 +122,8 @@ class AssistantCodeGenerator {
      * @private
      */
     _generateTAUWidgetVariable(options) {
-        let code = ((options.options && (Object.keys(options.options).length > 0)) 
-            ? Sentences.tauWidgetWithOptions.map(item => Mustache.render(item, options)).join(this._lineEnding)
+        const code = ((options.options && (Object.keys(options.options).length > 0)) 
```

Comment:
it can be written shorter: `(options.options && !Object.keys(options.options).length)`